app/controllers/api/v1/accounts/familiar_followers_controller.rb

```diff
@@ -12,14 +12,14 @@ def index
   private
 
   def set_accounts
-    @accounts = Account.without_suspended.where(id: account_ids).select('id, hide_collections').index_by(&:id).values_at(*account_ids).compact
+    @accounts = Account.without_suspended.where(id: account_ids).select('id, hide_collections').compact
```

Comment:
`compact` isn't required anymore!
```suggestion
    @accounts = Account.without_suspended.where(id: account_ids).select('id, hide_collections')
```